apps/frontend/src/app/page.tsx

```diff
@@ -1,10 +1,16 @@
 import { CardList } from '@/containers'
 import { SearchBar } from '@/components'
+import { Navbar } from '@/components/Navbar'
```

Comment:
import the `Navbar` component from the `index.ts` file inside the components folder, and import the component first in that `index` file